Authorization/Authorization/en.lproj/Localizable.strings

```diff
@@ -12,6 +12,9 @@
 "SIGN_IN.EMAIL_OR_USERNAME" = "Email or username";
 "SIGN_IN.PASSWORD" = "Password";
 "SIGN_IN.FORGOT_PASS_BTN" = "Forgot password?";
+"SIGN_IN.AGREEMENT" = "By signing in to this app, you agree to the [%@ End User License Agreement](%@) and [%@ Terms of Service and Honor Code](%@) and vou acknowledge that edX and each Member process your personal data in
```

Comment:
type `vou` it should be `you` and `edX` should be replaced platform name from config.